client/containers/App/SideMenu/ScopePicker.tsx

```diff
@@ -36,38 +24,6 @@ const ScopePicker = (props: Props) => {
 		icon = 'pubDoc';
 	}
 
-	const scopes: Scope[] = [];
```

Comment:
I guess this was all unnecessary (it got moved to `ScopeDropdown` at some point...) /shrug